ui/app/components/sidebar/nav/cluster.js

```diff
@@ -27,6 +27,21 @@ export default class SidebarNavClusterComponent extends Component {
     return this.namespace.inRootNamespace && !this.hasChrootNamespace;
   }
 
+  get canAccessVaultUsageDashboard() {
+    /*
+    A user can access Vault Usage if they satisfy the following conditions:
+      1) They have access to sys/v1/utilization-report endpoint
+      2) They are either
+        a) enterprise cluster and root namespace
+        b) hvd cluster and /admin namespace
+    */
+
+    return (
+      this.permissions.hasNavPermission('dashboard') &&
+      ((this.version.isEnterprise && this.isRootNamespace) || this.namespace.inHvdAdminNamespace)
```

Comment:
technically HVD managed clusters are enterprise so this could also be the following. either way works - whichever is clearer for folks. great comment above, thanks for being clear!
```suggestion
      this.version.isEnterprise && (this.isRootNamespace || this.namespace.inHvdAdminNamespace)
```

Reply:
**nitpick(non-blocking):** I always find it hard to follow when there are mixed logical operations in the same check, especially future me that has forgotten about all context related to this. I think breaking out the namespace check into it's own could make this a bit easier to reason about:

``` javascript
      const isCorrectNamespace = this.isRootNamespace || this.namespace.inHvdAdminNamespace;
      return this.permissions.hasNavPermission('dashboard') &&
      this.version.isEnterprise &&
      isCorrectNamespace;
```